test/multiverse/suites/excon/Envfile

```diff
@@ -9,13 +9,25 @@ excon_versions = [
   '0.64.0',
   '0.70.0',
   '0.78.1',
-  '0.85.0',
+  '0.85.0'
 ]
 
 def gem_list(excon_version = nil)
+  # TODO: CI - rack v3.0.0 causes 2 test failures
```

Comment:
Could you make an issue for us to address all these failures mentioned in the todos to make sure we don't lose track of it?

Reply:
it looks like you did that at the same time as I submitted the comment lol. thnak you!

Reply:
Done, that particular one is covered by #1393 which seems to impact all of the HTTP clients with new TODO statements, and the separate #1394 will track the new TODO for Unicorn.